pkg/apis/psmdb/v1/psmdb_defaults.go

```diff
@@ -232,6 +202,45 @@ func (cr *PerconaServerMongoDB) CheckNSetDefaults(platform version.Platform, log
 	}
 
```

Comment:
should we also add?
```
if cr.Spec.Sharding.ConfigsvrReplSet.Storage.Engine != WiredTiger {
    return errors.Errorf("%s storage engine is not supported for config server", cr.Spec.Sharding.ConfigsvrReplSet.Storage.Engine)
}
```

Reply:
https://github.com/percona/percona-server-mongodb-operator/pull/467/commits/a3d77d79a78b9eedca1a3d9b409f9101c4a427fb